pages/api/confirm.js

```diff
@@ -5,7 +5,7 @@ export default async function handler(req, res) {
     if (data.value=="Test Case Generated Successfully!!!"){
         await transporter.sendMail({
             from: data.email,
-            to:process.env.EMAIL,
+            to:process.env.EMAIL_SENDER,
```

Comment:
this was the issue @singhcoder694

Reply:
Ohh, env this is the reason on my system it was working properly , due to change in env variable name.... Okk👍

Reply:
No it should be like this

```
to: data.email,
from:process.env.EMAIL
```

And, we're using noreply mail ID. So, it getting bounced off.

Reply:
Got it👍